server/v2/stf/core_services.go

```diff
@@ -25,6 +33,14 @@ func (s storeService) OpenKVStore(ctx context.Context) store.KVStore {
 	return state
 }
 
+func (s storeService) OpenMemoryStore(ctx context.Context) store.KVStore {
+	return s.OpenKVStore(ctx)
+}
+
+func (s storeService) OpenTransientStore(ctx context.Context) store.KVStore {
+	return s.OpenKVStore(ctx)
+}
```

Comment:
just a heads up -- we don't have a notion of transient KVStore in store/v2.

Reply:
What happens for modules using it then? 👀

Reply:
we could use a normal kvstore under the hood, some teams may have not migrated out of it